packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -554,17 +561,27 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     }
   });
 
+  const resizeObserverRef = React.useRef(null);
+
   React.useEffect(() => {
     const handleResize = debounce(() => {
       updateIndicatorState();
       updateScrollButtonState();
     });
 
-    const win = ownerWindow(tabsRef.current);
-    win.addEventListener('resize', handleResize);
+    try {
+      resizeObserverRef.current = new ResizeObserver(handleResize);
+    } catch (err) {
```

Comment:
I think that we could:

1. Keep the resize handler for the browsers that don't support the resize observer API
2. Not use try catch but test if the API is available with a straight if.

Reply:
1. Addressed.
2. This also prevents test failure, so I think we still need to include it?